test/e2e/replica_set_arbiter/replica_set_arbiter_test.go

```diff
@@ -69,9 +69,9 @@ func TestReplicaSetArbiter(t *testing.T) {
 			resourceName:     "mdb4",
 		},
 	}
-	for testName, _ := range tests {
```

Comment:
is the linter complaining over explicit empty assigns?

Reply:
Yeah. it is. I'm not sure why but apparently it doesn't like when you leave the second one empty in a `for`. Kinda makes sense I guess.